src/scenes/ProgressReports/StatusHistory/StatusHistory.tsx

```diff
@@ -0,0 +1,117 @@
+import { ProjectBreadcrumb } from '~/components/ProjectBreadcrumb';
+import { Navigate } from '~/components/Routing';
+import { Error } from '../../../components/Error';
+import {
+  ProgressReportDetailDocument,
+  ProgressReportDetailFragment,
+} from '../Detail/ProgressReportDetail.graphql';
+import { WorkFlowEventHistory } from '../Detail/WorkflowEventsHistory';
+
+export const StatusHistory = () => {
+  const { id, changesetId } = useChangesetAwareIdFromUrl('reportId');
+  const [checked, set] = useToggle();
+
+  const { data, error } = useQuery(ProgressReportDetailDocument, {
+    variables: {
+      id,
+      changesetId,
+    },
+  });
+  if (error) {
+    return (
+      <Error page error={error}>
+        {{
+          NotFound: 'Could not find status history',
+          Default: 'Error loading status history',
+        }}
+      </Error>
+    );
+  }
+
+  const report = data?.periodicReport as
+    | ProgressReportDetailFragment
+    | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- safety check. It's possible with manual user input.
+  if (report && report.__typename !== 'ProgressReport') {
+    return <Navigate to="/" replace />;
+  }
+
+  const engagement = report?.parent;
+
+  return (
+    <div css={{ overflowY: 'auto' }}>
+      <Box component="main" sx={{ ...flexColumnStyle, p: 4 }}>
+        <Helmet title="Status History" />
+        <Breadcrumbs
+          children={[
+            <ProjectBreadcrumb key="project" data={engagement?.project} />,
+            <EngagementBreadcrumb key="engagement" data={engagement} />,
+            <Breadcrumb
+              key="report-list"
+              to={
+                engagement
+                  ? `/engagements/${idForUrl(engagement)}/reports/progress`
+                  : undefined
+              }
+            >
+              {!report ? <Skeleton width={200} /> : 'Quarterly Reports'}
+            </Breadcrumb>,
+            <Breadcrumb
+              key="report"
+              to={
+                engagement ? `/progress-reports/${idForUrl(report)}` : undefined
+              }
+            >
+              {!report ? (
+                <Skeleton width={200} />
+              ) : (
+                <ReportLabel report={report} />
+              )}
+            </Breadcrumb>,
+            <Breadcrumb key="status-history" to=".">
+              {!report ? <Skeleton width={200} /> : 'Status History'}
+            </Breadcrumb>,
+          ]}
+        />
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+            pt: 2,
+          }}
+        >
+          <Typography variant="h3" paragraph sx={{ pt: 2 }}>
+            Status History
+          </Typography>
+          <FormControlLabel
+            control={
+              <>
+                <Typography sx={{ pr: 1 }}>Hide Notes</Typography>
```

Comment:
I would not have this toggle be a negative. Instead I would say "Show Notes" and default it according to whether we want them shown or hidden by default.

Also why is this here instead of in the `label` prop?
...Oh it's for placement. Instead you can use `labelPlacement="start"`.
If you are looking for more spacing between the switch, then `sx={{ gap: 1 }}` will do the trick.

I'd keep the switch the default size as well. We are not strapped for space here.